src/strategies/bearerTokenStrategy.ts

```diff
@@ -58,11 +59,19 @@ const bearerTokenStrategy = (
 };
 
 function fulfill(
-  errorOrUser: Either<Error, User>,
+  errorOrUser: Either<Error, Option<User>>,
   // tslint:disable-next-line:no-any
-  done: (error: any, user?: any, options?: IVerifyOptions | string) => void
+  done: (error: any, user?: any, options?: IVerifyOptions | string) => void,
+  scope: string
```

Comment:
why is `token` above but `scope` here? is it the same thing?

Reply:
It was because of `IVerifyOptions`  require scope parameters with a message. But I sow the implementation of `passport-http-bearer` and when options is a string it was the same that passes only the message. Fixed in 0511e0e